CONTRIBUTING.md

```diff
@@ -226,3 +226,5 @@ please read [GitHub's guides](https://docs.github.com/en/github/collaborating-wi
 
 By contributing to Torchvision, you agree that your contributions will be licensed
 under the LICENSE file in the root directory of this source tree.
+
+Contributers are also required to [sign our Contributor License Agreement](https://code.facebook.com/cla). If you are contributing on behalf of someone else (eg your employer), the individual CLA may not be sufficient and your employer may need to sign the corporate CLA. If you have any questions, please contact us at cla@meta.com
```

Comment:
Since the process (individual vs corporte CLA) is clearly described immediately in the provided link, we can probably keep it short here?

```suggestion
Contributors are also required to [sign our Contributor License Agreement](https://code.facebook.com/cla).
```